includes/pointers.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Notes in a Pointer dialog box for guiding users in the dashboard interface.
+ */
+add_action( 'admin_enqueue_scripts', 'pmpro_enqueue_admin_scripts' );
+/**
+ * [pmpro_enqueue_admin_scripts] Enqueue the scripts needed to builder admin pointers.
+ * 
+ * @return void
+ */
+function pmpro_enqueue_admin_scripts() {
+	wp_enqueue_style( 'wp-pointer' );
+	wp_enqueue_script( 'wp-pointer' );
+	// hook the pointer
+	add_action( 'admin_print_footer_scripts', 'prepare_pointer_script' );
+}
+/**
+ * [prepare_pointer_script] Details about PMPro 2.0 that are added to the Admin Pointer
+ * 
+ * @return void
+ */
+function prepare_pointer_script() {
```

Comment:
pmpro_prepare_pointer_scripts

Reply:
I agree that we want to prefix this, but want to add that since this is going into core, a clone of this function can be used for any Add On that wants a Pointer.